docs/remote-signing.md

```diff
@@ -118,6 +130,10 @@ remotesigner.enable=true
 remotesigner.rpchost=zane.example.internal:10019
 remotesigner.tlscertpath=/home/watch-only/example/signer.tls.cert
 remotesigner.macaroonpath=/home/watch-only/example/signer.custom.macaroon
+# Optionally, specify that the watch-only node uses an inbound remote signer.
+# However, since the default signerrole is "watchonly-inbound," this isn't
+# required.
+remotesigner.signerrole=inbound
```

Comment:
nit: I think this needs to say `watchonly-inbound`.

Reply:
Ah nice, good catch! Thanks